### ISSUE TITLE
Investigate Max CFI Metrics

### ISSUE THREAD

**Issue by dfitchett** [department-of-veterans-affairs/abd-vro#2160]
## Summary 
Investigate what happens with the max cfi metric in the code base.

## Associated Items
slack [thread](https://dsva.slack.com/archives/C01BVF2A3V0/p1698187706277159)

## Questions to answer: 
Thoughts on why the # of Veterans in the control group seem to consistently be larger than the # Veterans in the experiment group: 

1. Confirm: On the screen within the 526 application that prompts the Veteran to choose whether they are submitting a new or increase claim, our metric is logged when they select the increase checkbox and click continue IF the Veteran has a 10% tinnitus rating.
2. what if...someone goes back to the previous screen (the one before the new/increase question), then decided to move forward again and re-checked the increase box? this would log another metric, but this could happen for either groups. doesn't explain the difference.
3. what if...someone saw the education and abandoned the form? we wouldn't log another metric in this scenario.
4. what if...someone went back and changed their response to "new"? a metric wouldn't be logged, because they didn't click increase.
5. what if...someone chose to continue an existing 526 application instead of starting a new one? the metric wouldn't get logged because the rating education is only added to the form on new applications.

**Comment by dfitchett** [comment]
Answers: 
1. Confirmed behavior is `CFI In Progress` metric is logged when the user sees this page **AND** has checked the "One or more of my rated conditions that have gotten worse" **ONLY IF** the user has tinnitus rated at 10% and has started a form since we launched.
<img src="https://github.com/department-of-veterans-affairs/abd-vro/assets/135860892/664be53c-9ce9-4632-b8aa-b9019ae3cfa1" width= "500">

2. This would not log the `CFI In Progress` metric metric again, the metadata stored once they clicked it the first time prevents it from being logged again.
3. We would log the `CFI In Progress` metric once, but would never log the `CFI Submitted` metric.
4. If someone changed their response to new after having checked the "One or more of my rated conditions that have gotten worse" checkbox, the metric would have already been logged once, and would not be affected
5. The meta data that prevents logging the metric multiple times is stored in the database with in progress form meaning...
  -  if they reached the point where the metric was logged, then opted to "Finish this application later", then went to continue the application, the metric would not be logged again. 
  - if they did not reach the point where the metric was logged, then opted to "Finish this application later", then went to continue the application, the metric would only be logged if they reached the conditions in answer 1.

**Comment by dfitchett** [comment]
- [ ] Verify this same behavior in staging.

**Comment by amylai-nava** [comment]
^I checked the checkbox since we verified the same behavior in staging on Friday, 11/3 with a test user that has a 0% tinnitus rating and a test user that has a 10% tinnitus rating.

Just noting that we didn't test the scenario where a Veteran has a 10% tinnitus rating + other rated disabilities, **and** submits a claim for non-tinnitus disabilities only. Given the other testing we did, we didn't think testing this scenario would be useful.
